src/test/scala/uk/gov/nationalarchives/tdr/api/service/AntivirusMetadataServiceSpec.scala

```diff
@@ -51,4 +51,38 @@ class AntivirusMetadataServiceSpec extends AnyFlatSpec with MockitoSugar with Ma
     result.result shouldBe "result"
     result.datetime shouldBe dummyInstant.toEpochMilli
   }
+
+  "getAntivirusMetadata" should "call the repository with the correct arguments" in {
+    val avRepositoryMock = mock[AntivirusMetadataRepository]
+    val consignmentCaptor: ArgumentCaptor[UUID] = ArgumentCaptor.forClass(classOf[UUID])
+    val consignmentId = UUID.randomUUID()
+    when(avRepositoryMock.getAntivirusMetadata(consignmentCaptor.capture())).thenReturn(Future(Seq()))
+    new AntivirusMetadataService(avRepositoryMock).getAntivirusMetadata(consignmentId).futureValue
+    consignmentCaptor.getValue should equal(consignmentId)
+  }
+
+  "getAntivirusMetadata" should "return the correct data" in {
+    val fixedFileUuid = UUID.fromString("07a3a4bd-0281-4a6d-a4c1-8fa3239e1313")
+    val dummyTimestamp = Timestamp.from(FixedTimeSource.now)
+    val consignmentId = UUID.randomUUID()
+    val avRepositoryMock = mock[AntivirusMetadataRepository]
+    val mockResponse = Future.successful(Seq(AvmetadataRow(
+      fixedFileUuid,
+      "software",
+      "software version",
+      "database version",
+      "result",
+      dummyTimestamp
+    )))
+
+    when(avRepositoryMock.getAntivirusMetadata(any[UUID])).thenReturn(mockResponse)
```

Comment:
Maybe use the actual consignmentId instead of an argument matcher: `any[UUID]`

Reply:
Yep, makes sense, I've changed it.